### PR TITLE
Add constraint for prisma schema

### PR DESCRIPTION
**Pull request by kachick**
How about to add some constraints in DB layer?
`Nullable completed flag`, `Null texts`, `Duplicated texts` are basically not needed in TODO app, I think.

Before
---

```
postgres=# \d "Task";
                               Table "public.Task"
  Column   |  Type   | Collation | Nullable |              Default
-----------+---------+-----------+----------+------------------------------------
 id        | integer |           | not null | nextval('"Task_id_seq"'::regclass)
 text      | text    |           |          |
 completed | boolean |           |          | false
Indexes:
    "Task_pkey" PRIMARY KEY, btree (id)
```

After
---

```
postgres=# \d "Task";
                               Table "public.Task"
  Column   |  Type   | Collation | Nullable |              Default
-----------+---------+-----------+----------+------------------------------------
 id        | integer |           | not null | nextval('"Task_id_seq"'::regclass)
 text      | text    |           | not null |
 completed | boolean |           | not null | false
Indexes:
    "Task_pkey" PRIMARY KEY, btree (id)
    "Task.text_unique" UNIQUE, btree (text)
```

### CODE REVIEW

**Comment by mattn** [comment]
I wonder this change will break go client.

**Comment by kachick** [comment]
😮 Sorry, I'll check 🙇

File: README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ $ go generate
 $ go build
 $ echo 'DATABASE_URL=postgresql://yourname:yourname@localhost:5432/postgres' > .env
 $ npx prisma db push --preview-feature
-$ ./go-prisma-example
+$ DATABASE_URL=postgresql://yourname:yourname@localhost:5432/postgres ./go-prisma-example
```
Comment (kachick) [inline README.md:15]: When I trying to run, `.env` does not works in build binary 🤔

**Comment by kachick** [comment]
You're right. 

```console
$ npx prisma generate
$ go generate
```

It makes compile error 🙇 

After applying https://github.com/mattn/go-prisma-example/pull/1/commits/9a4e0346d910eb875269e73d0af04e3d292a2547, I can rebuild and run. Could you re-review?

File: main.go [hunk 2]
```diff
@@ -35,14 +35,8 @@ func main() {
...
-		var text *string
-		if newText, ok := task.Text(); ok {
-			text = &newText
-		}
-		var completed *bool
-		if newCompleted, ok := task.Completed(); ok {
-			completed = &newCompleted
-		}
+		text := &task.Text
+		completed := &task.Completed
```
Comment (mattn) [inline main.go:39]: This does not work.

https://github.com/prisma/prisma-client-go/blob/33b5e2c53b2bcbbe4a7f6f8cac5b8653c6c4e521/docs/reference/15-if-present-methods.md#querying-dynamically
>but if a variable is nil, it means that that field should be ignored.

Comment (kachick) [reply main.go:39]: 😮 Thank you for letting me know!
🙇 I should read the reference...

**Comment by kachick** [comment]
Sorry to keep draft state in longtime 🙇‍♂️  Please let me close for now

**Pull request closed without merging**